src/Whitelist/VulnerabilityExcluder.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { existsSync, readFileSync } from "fs";
+import path from 'path';
+
+import { OssIndexServerResult, Vulnerability } from "../Types/OssIndexServerResult";
+import { Whitelist } from "../Types/Whitelist";
+
+const whitelistFilePathPwd = path.join(process.cwd(), 'auditjs.json');
+
+export abstract class VulnerabilityExcluder {
```

Comment:
No need to think of every single thing in terms of classes.  You could just export the `filterVulnerabilities` function itself directly.

Reply:
I think I left it as a class so I could send it the `Logger`, for ease of testing later (send in a dummy logger), etc...